docs/ruleset_functions/get_heating_fuels_p.md

```diff
@@ -0,0 +1,75 @@
+# get_heating_fuels_p
+
```

Comment:
Function replaced by a function that does not solely apply to proposed and that expanded the checks.